deno/lib/common.ts

```diff
@@ -0,0 +1,47 @@
+import { load } from "https://deno.land/x/eszip@v0.18.0/loader.ts";
```

Comment:
This file was copied untouched from the edge bootstrap repo.

Some of the functions here are used by the edge bootstrap when generating the stage 1 ESZIP. We can live with the duplicated code for now, and later on we can change edge bootstrap so that the files are loaded from this repo instead. Conceptually, it makes more sense for the bootstrap to load bundling logic from the bundler repo than the other way around.